pkg/ctl/install/fluxinstaller.go

```diff
@@ -0,0 +1,558 @@
+package install
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"time"
+
+	helmopinstall "github.com/fluxcd/helm-operator/pkg/install"
+	"github.com/kris-nova/logger"
+	"github.com/pkg/errors"
+	"github.com/riywo/loginshell"
+	fluxinstall "github.com/weaveworks/flux/install"
+	corev1 "k8s.io/api/core/v1"
+	kubeclient "k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
+	tillerinstall "k8s.io/helm/cmd/helm/installer"
+	"sigs.k8s.io/yaml"
+
+	"github.com/weaveworks/eksctl/pkg/ctl/cmdutils"
+	"github.com/weaveworks/eksctl/pkg/git"
+	"github.com/weaveworks/eksctl/pkg/kubernetes"
+)
+
+const (
+	fluxNamespaceFileName    = "flux-namespace.yaml"
+	tillerManifestPrefix     = "tiller-"
+	tillerNamespaceFileName  = tillerManifestPrefix + "namespace.yaml"
+	tillerServiceName        = "tiller-deploy" // do not change at will, hardcoded in Tiller's manifest generation
+	tillerServiceAccountName = "tiller"
+	tillerRBACTemplate       = `apiVersion: v1
+kind: ServiceAccount
```

Comment:
At very least please put into a separate file and use bindata. The convention is to have such files in `assets` subdirectory of a package.

Reply:
It's overkill for a single file which nicely in the code.